tests/test_choose_disk.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2017, Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import imp
+import inspect
+import mock
+import os
+from nose.tools import assert_equals, nottest
+from library import choose_disk
+
+host_dir = "tests/unit/hosts/"
+
+
+@nottest
+def get_hosts_to_test():
```

Comment:
another bonus of using `py.test` is that this wouldn't need to exist as there is support for fixtures to be loaded per test function (or test method)